src/main/java/net/minecraftforge/unification/UnificationConstants.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.unification;
+
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.item.EnumDyeColor;
+import net.minecraft.item.ItemStack;
+
+/**
+ * Provides a list of predefined {@link UnifiedMaterial} and {@link UnifiedForm} for convenience.
+ */
+public final class UnificationConstants
+{
+    public static final UnifiedMaterial IRON = UnificationManager.getMaterial("iron");
+    public static final UnifiedMaterial GOLD = UnificationManager.getMaterial("gold");
+    public static final UnifiedMaterial DIAMOND = UnificationManager.getMaterial("diamond");
+    public static final UnifiedMaterial LAPIS = UnificationManager.getMaterial("lapis");
+    public static final UnifiedMaterial REDSTONE = UnificationManager.getMaterial("redstone");
+    public static final UnifiedMaterial QUARTZ = UnificationManager.getMaterial("quartz");
+    public static final UnifiedMaterial COAL = UnificationManager.getMaterial("coal");
+    public static final UnifiedMaterial EMERALD = UnificationManager.getMaterial("emerald");
+
+    public static final UnifiedForm ORE = UnificationManager.getForm("ore");
+    public static final UnifiedForm INGOT = UnificationManager.getForm("ingot");
+    public static final UnifiedForm DUST = UnificationManager.getForm("dust");
+    public static final UnifiedForm NUGGET = UnificationManager.getForm("nugget");
+    public static final UnifiedForm GEM = UnificationManager.getForm("gem");
+    public static final UnifiedForm BLOCK = UnificationManager.getForm("block");
+
```

Comment:
Can you add wood logs here? We can replace `Block#isWood` with this unification system.

Reply:
This is where we begin to enter strange territory and big debates can happen.
For this PR, I want to draw the line at ores and avoid the issue.

Reply:
I would argue that logs -> planks is a standard unification, both of those could be added, but I wouldn't add doors/fences etc. (Although they could be) I guess that is the debate.